src/test/java/com/bitmovin/api/examples/CreateEncodingWithBurnInTextFilter.java

```diff
@@ -0,0 +1,454 @@
+import com.bitmovin.api.encoding.encodings.muxing.TSMuxing;
+import com.bitmovin.api.encoding.encodings.streams.Stream;
+import com.bitmovin.api.encoding.enums.CloudRegion;
+import com.bitmovin.api.encoding.enums.DashMuxingType;
+import com.bitmovin.api.encoding.enums.StreamSelectionMode;
+import com.bitmovin.api.encoding.filters.TextFilter;
+import com.bitmovin.api.encoding.filters.enums.TextFilterFont;
+import com.bitmovin.api.encoding.inputs.HttpsInput;
+import com.bitmovin.api.encoding.manifest.dash.AdaptationSet;
+import com.bitmovin.api.encoding.manifest.dash.AudioAdaptationSet;
+import com.bitmovin.api.encoding.manifest.dash.DashFmp4Representation;
+import com.bitmovin.api.encoding.manifest.dash.DashManifest;
+import com.bitmovin.api.encoding.manifest.dash.Period;
+import com.bitmovin.api.encoding.manifest.dash.VideoAdaptationSet;
+import com.bitmovin.api.encoding.manifest.hls.HlsManifest;
+import com.bitmovin.api.encoding.manifest.hls.MediaInfo;
+import com.bitmovin.api.encoding.manifest.hls.MediaInfoType;
+import com.bitmovin.api.encoding.manifest.hls.StreamInfo;
+import com.bitmovin.api.encoding.outputs.Output;
+import com.bitmovin.api.encoding.outputs.S3Output;
+import com.bitmovin.api.encoding.status.Task;
+import com.bitmovin.api.enums.Status;
+import com.bitmovin.api.examples.util.H264Representation;
+import com.bitmovin.api.exceptions.BitmovinApiException;
+import com.bitmovin.api.http.RestException;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CreateEncodingWithBurnInTextFilter
+{
+    private static String ApiKey = "<INSERT YOUR API KEY>";
+
+    private static CloudRegion cloudRegion = CloudRegion.AWS_EU_WEST_1;
+    private static String HTTPS_INPUT_HOST = "input.host.com";
+    private static String HTTPS_INPUT_PATH = "path/to/your/input/file.mkv";
+
+    private static String S3_OUTPUT_ACCESSKEY = "<YOUR S3 OUTPUT ACCESS KEY>";
+    private static String S3_OUTPUT_SECRET_KEY = "<YOUR S3 OUTPUT SECRET KEY>";
+    private static String S3_OUTPUT_BUCKET_NAME = "<YOUR OUTPUT BUCKET>";
+    private static String OUTPUT_BASE_PATH = "/your/output/base/path";
+
+    private List<H264Representation> h264Representations = new ArrayList<H264Representation>() {{
+        add(new H264Representation(240, null, null, 400000L , ProfileH264.HIGH, OUTPUT_BASE_PATH, "video/240p"));
+        add(new H264Representation(360, null, null, 800000L, ProfileH264.HIGH, OUTPUT_BASE_PATH, "video/360p"));
+        add(new H264Representation(480, null, null, 1200000L, ProfileH264.HIGH, OUTPUT_BASE_PATH, "video/480p"));
+        add(new H264Representation(720, null, null, 2400000L, ProfileH264.HIGH, OUTPUT_BASE_PATH, "video/720p"));
+        add(new H264Representation(1080, null, null, 4800000L, ProfileH264.HIGH, OUTPUT_BASE_PATH, "video/1080p"));
```

Comment:
Specifying the width is good! However, you used the height values, e.g., 1080 instead of 1920. Please use the width values here instead of the height values.